src/bundle/Resources/public/scss/_multi-collapse.scss

```diff
@@ -0,0 +1,37 @@
+.ibexa-multi-collapse {
```

Comment:
Moved styles from product-catalog. Based on previous review comment let me know if I still should change class name to ibexa-pc-...